core/src/main/kotlin/org/eln2/sim/electrical/mna/Circuit.kt

```diff
@@ -468,11 +468,11 @@ class Circuit {
      * This routine can fail; [success] is set based on whether this method was successful.
      */
     // Step 3: With known current and voltage sources, solve for unknowns (node potentials and source currents).
-    private fun computeResult() {
+    internal fun computeResult(): RealVector? {
```

Comment:
I added a return here so that you can dump the `unknowns` matrix out of this and into a testing class.